mobile/README.md

```diff
@@ -46,6 +89,8 @@ To checkout the latest tagged release of lnd, run
 
 ### Building `lnd` for Android
 
+Go to `$GOPATH/src/github.com/lightningnetwork/lnd` and run the command below (make sure that the Docker is running):
```

Comment:
I assume you mean "the Docker engine" here?

Reply:
I run the Docker desktop app before running this command. Is it Docker engine?

Reply:
Yes, I think it's called the docker engine or daemon. It could also just say "make sure Docker is running", just "the Docker" sounded weird.